spec/unit/primitive_consul_health_spec.rb

```diff
@@ -21,6 +21,7 @@
               'service:ping' => {
                 CheckID: 'service:ping',
                 Name: "Service 'ping' check",
+                ServiceName: 'service',
```

Comment:
Should we add an explicit test on this ?

Reply:
done